presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/file/FileHiveMetastoreConfig.java

```diff
@@ -32,9 +32,10 @@ public String getCatalogDirectory()
 
     @Config("hive.metastore.catalog.dir")
     @ConfigDescription("Hive file-based metastore catalog directory")
-    public void setCatalogDirectory(String catalogDirectory)
+    public FileHiveMetastoreConfig setCatalogDirectory(String catalogDirectory)
```

Comment:
We could make this `org.apache.hadoop.fs.Path`

Reply:
let's consider this follow up